core/src/main/resources/swagger/api-docs.yaml

```
@@ -5625,7 +5625,27 @@ components:
             Sam.
         managedAppCoordinates:
           $ref: '#/components/schemas/AzureManagedAppCoordinates'
+        members:
+          description: The list of members to add to the billing project
+          type: array
+          items:
+            $ref: '#/components/schemas/ProjectAccessUpdate'
+        inviteUsersNotFound:
+          description: If true, invite the users to Terra if they are not registered in the system. They will be given access to the project
```

Nit: it looks like this sentence is unfinished?

```suggestion
          description: If true, invite the users to Terra if they are not registered in the system. They will be given access to the project when they register.
```